### DISCUSSION TITLE
How do I automate tampermonkey with webpack

### DISCUSSION THREAD

**Discussion by Murka007** [Tampermonkey/tampermonkey#1562] [Q&A]
I wanted to automate the whole development process with tampermonkey and webpack. How do I achieve this?

**Comment by 7nik** [comment]
You can try [webpack-userscript](https://github.com/momocow/webpack-userscript) plugin. Though I could use it, if I remember right, due to conflicts of required packages with other libs.

Also, you can use a more manual approach - use a lib to generate the userscript header and add it at the beginning of the output file. [Here](83c6dde3c72accf22ddd61260737af8aeffd560e) is an example of how I did it.
